cmd/changelog-parser/testdata/repositories.yml

```diff
@@ -13,14 +13,17 @@ section:
         version: v1.4.6
         after: v1.3.5
         upgrade_url: https://docs.cyberark.com/Product-Doc/OnlineHelp/AAM-DAP/Latest/en/Content/Deployment/Upgrade/upgrade-intro.htm
+        certification: "trusted"
       - name: cyberark/conjur-oss-helm-chart
         url: https://github.com/cyberark/conjur-oss-helm-chart
         description: Helm chart for deploying Conjur OSS.
         version: v1.3.7
+        certification: "certified"
```

Comment:
This should be trusted

Reply:
Ditto - also just test data for the integration test